runtime/lua/vim/lsp.lua

```diff
@@ -933,7 +933,7 @@ function lsp.status()
   end
   local message = table.concat(messages, ', ')
   if percentage then
-    return string.format('%03d: %s', percentage, message)
+    return string.format('%.0f%%: %s', percentage, message)
```

Comment:
```suggestion
    return string.format('%3d%: %s', percentage, message)
```

The idea behind the 03d was to avoid shifting when it changes from single, to double to 3 digits. If the `0` is not nice, maybe a whitespace is okay?

Reply:
should be `%3d%%` . whitespace maybe no need. just a test it works fine on stl

https://github.com/neovim/neovim/assets/41671631/2976830b-c3a9-4558-adf1-fd47458d921e